public/test/examples/cra/src/App.js

```diff
@@ -28,7 +28,7 @@ function App() {
       console.error(new Error("Baz"));
 
       // eslint-disable-next-line no-undef
-      dump(`RecReplaySendAsyncMessage Example__Finished`);
+      console.log("Example__Finished");
```

Comment:
It seems like this weird string can be changed to something more appropriate now, or is there a better way to signal to playwright that the test has finished?

Reply:
there aren't any great mechanisms for the webpage to talk to playwright and it's probably something that should be discouraged. Our tests and examples are a bit weird in this regard.

We probably could re-write our examples so that playwright decides when they are done.
At somepoint, we might might want to re-write these tests so that they are also run through playwright as opposed to being loaded in content. This doesn't seem like a priority for the time being though.